.github/ISSUE_TEMPLATE/02-package-request.yml

```diff
@@ -24,13 +24,42 @@ body:
       placeholder: Link to Git or source archive
     validations:
       required: true
+
   - type: checkboxes
     attributes:
       label: Packaging policy acknowledgement
-      description: We reject packages that are used for Hacking or fall under certain other conditions described in [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy). Please READ this policy document!
+      description: |
+        We reject packages that are used for Hacking or fall under certain other conditions described in [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).
+        Please ***READ*** this policy document for the specific meanings of the terms below!
       options:
-        - label: I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied in case of violation.
+        - label: The project is actively developed.
+            ###### We prefer packaging actively maintained projects
+          required: false
+        - label: The project has [existing packages](https://repology.org/projects) and is "well known".
+            ###### We prefer packaging projects with [existing packages on other distributions](https://repology.org/projects).
+          required: false
+        - label: Licensed under a [widely recognized open source license](https://opensource.org/license).
+            ###### SSPL-1.0 and BSL-1.1 specifically are ***not*** suitable for Termux packages.
```

Comment:
Why not?

Reply:
That's HashiCorp (terraform) and ElasticSeach's "source available" licenses in particular.
They are incompatible with the GPL, BSD, MIT, Apache, etc licenses.

In short, both limit user freedoms to an obstructive amount, and we've decided to not deal with either.
They're perfectly fine for packaging in the TUR if the package is still desired.

BSL-1.1 is the weirder of the two cases as it has a 5 year "succession" clause, whereby the licensor may designate a OSS license to apply to any releases of the software older than 5 years.
But shipping 5 year out of date software does not remedy the initial license.

Reply:
> That's MongoDB and HashiCorp's (terraform, vagrant) "source available" licenses in particular.
They are incompatible with the GPL, BSD, MIT, Apache, etc licenses.

I know what they are, the GPL is incompatible with most OSS licenses also.

> In short, both limit user freedoms to an obstructive amount, and we've decided to not deal with either.

When was this decided and by whom? The SSPL is even more viral than the AGPL, how is that limiting user freedom?

> BSL-1.1 is the weirder of the two cases as it has a 5 year "succession" clause, whereby the licensor may designate a OSS license to apply to any releases of the software older than 5 years.
But shipping 5 year out of date software does not remedy the initial license.

Nobody is suggesting shipping old software. I don't see any reason to exclude these new licenses that provide all source, just to follow the FSF/OSI priests, who frankly are fairly dumb IMO.

Reply:
I'll drop the comment.
I can't actually find the discussion, I'm not sure we ever formalized it,
and I shouldn't be formalizing stuff without consensus.

Reply:
We have list of accepted licenses in the package linter.
https://github.com/termux/termux-packages/blob/6ffa25fcbab87bd3ae2b1f2a7abbbb078a350a4e/scripts/lint-packages.sh#L18-L36
I'd just prefer not to throw that at users, we could also link to `choosealicense` alternatively, but they show a much smaller selection of licenses.
I guess https://spdx.org/licenses/ would be a "neutral" choice.